workflows/surrogate_models/1d_loglog_pce/example_numerical_parameters.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+import boutvecma
+import easyvvuq as uq
+import chaospy
+import os
+import numpy as np
+import time
+import matplotlib.pyplot as plt
+
+# Do one run with atol=-15 to find value to use as offset in error decoder
+encoder = boutvecma.BOUTExpEncoder(
+    template_input="../../../models/conduction/data/BOUT.inp"
+)
+decoder = boutvecma.SimpleBOUTDecoder(variables=["T"])
```

Comment:
Should this be the log-abs version?

Reply:
It can't be the abs version because it's now using the exponent, which can be negative. I could have used the non-abs log version, but doing it this way lets the user plot both the log and non-log version.